bridgestyle/arcgis/togeostyler.py

```diff
@@ -434,6 +448,16 @@ def _extractFillColor(symbolLayers):
             return color
     return "#000000"
```

Comment:
Since the default color for the stroke is black, maybe it's a better idea to have white as the default fill color? Apart from this minor thing, it all looks good to me